CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## 7.2.0
+
+### Noticeable changes
+
+- Action creators can now have the signature `action(params,
+  callback)`. This is an error-first callback (like node). If an error
+  is provided, it will not dispatch an action. See below about this breaking update.
+- Stores now contain the logic to determine if they should respond to
+  an action. This is defined in `willRespondTo`.
+- Stores now contain the logic for how it transforms values. This
+  logic is contained under `transform`.
+- Added `getInitialState` method to Microcosm. By default, it will
+  return the reduced result of calling `getInitialState` on stores.
+- Added `reset` method to Microcosm. It will set an instance's state
+  to the result of calling `getInitialState`
+- `Microcosm::deserialize` will now only operate on the keys provided
+  by the seed object. This means that data passed into `replace`
+  will only blow way keys provided in the data object.
+
+### Breaking Changes
+
+Actions will follow the error-first callback convention if more than one argument is provided to them. This means that all configuration for an action should be passed as a single argument. If you are using multiple arguments in your actions, consider changing something like:
```

Comment:
Maybe link "the error-first callback convention" to http://thenodeway.io/posts/understanding-error-first-callbacks/